core/cat/factory/llm.py

```diff
@@ -60,6 +60,22 @@ class Config:
             "description": "Chat model from OpenAI",
         }
 
+class LLMAzureChatOpenAIConfig(LLMSettings):
+    openai_api_key: str
+    model_name: str = "gpt-4"
+    openai_api_base: str
+    api_type: str = "azure"
+    # https://learn.microsoft.com/en-gb/azure/cognitive-services/openai/reference#chat-completions
+    api_version: str = "2023-03-15-preview"
+    deployment_name: str
+
+    _pyclass: PyObject = langchain.chat_models.AzureChatOpenAI
```

Comment:
@pieroit do you know why langchain has a `langchain.llms.OpenAIChat` model but for azure it is a `langchain.chat_models.AzureChatOpenAI` ?

Reply:
@pieroit you missed this comment from 2 weeks ago